package.json

```diff
@@ -57,5 +62,8 @@
   "bugs": {
     "url": "https://github.com/gkjohnson/three-gpu-pathtracer/issues"
   },
-  "homepage": "https://github.com/gkjohnson/three-gpu-pathtracer#readme"
+  "homepage": "https://github.com/gkjohnson/three-gpu-pathtracer#readme",
+  "alias": {
+    "buffer": false
+  }
```

Comment:
Parcel tries to automatically install [buffer](https://www.npmjs.com/package/buffer), and that messes some things up here, because my packages contain some lines like this:

```js
if ( typeof Buffer !== 'undefined' ) {
  // ...
}
```

So this just tells Parcel to ignore it.